autoconnect/autoconnect-web/src/error.rs

```diff
@@ -25,7 +25,8 @@ impl ResponseError for ApiError {
 
     fn error_response(&self) -> HttpResponse {
         let code = self.status_code();
-        HttpResponse::build(code).json(json!({
+        let mut builder = autopush_common::errors::build_error(code);
```

Comment:
Can we keep these changes exclusive to #476? I briefly looked at it yesterday and had some thoughts on how much should even apply to autoconnect